### ISSUE TITLE
Staging Review finding: Use the VA/USA grid system and shorten character text line length

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#93671]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Transition Experience
**Product:** Transition Experience MVP v1
**Feature:** V1 Transition Experience

## Findings details
**VA.gov Experience Standard - issue:** [User experience is not consistent with the VA.gov design principles.](https://design.va.gov/about/experience-standards/#users-experience-is-consistent-with-the-vagov-design-principles)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @allison0034 (Design), @briandeconinck (Accessibility)

### Description

Most of your pages seem to be full width of the header. [Our guidance](https://design.va.gov/foundation/layout/float-grid#guidance) says to avoid using text lines that are longer than 75 characters and with your content spanning so long it is violating this guidance.

<details><summary>Link, screenshot or steps to recreate</summary>

![screen shot](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/82863-Allison-1.png?raw=true)

</details>

## Recommended action

Check your code and make sure you are using the VA/USA grid system we have in the foundation section of the Design System. One way to do this is to use the css class usa-width-two-thirds.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** [Layout](https://design.va.gov/foundation/layout/)
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by nkontrabecki** [comment]
Completed. 12/3/2024.
